rpc/internal/system_metrics.ext.go

```diff
@@ -0,0 +1,29 @@
+package internal
+
+import (
+	"github.com/evergreen-ci/cedar/model"
+)
+
+// Export exports SystemMetricsInfo to the corresponding SystemMetricsInfo
+// type in the model package.
+func (sm SystemMetricsInfo) Export() model.SystemMetricsInfo {
+	return model.SystemMetricsInfo{
+		Project:   sm.Project,
+		Version:   sm.Version,
+		Variant:   sm.Variant,
+		TaskName:  sm.TaskName,
+		TaskID:    sm.TaskId,
+		Execution: int(sm.Execution),
+		Mainline:  sm.Mainline,
+	}
+}
+
+// Export exports SystemMetricsArtifactInfo to the corresponding SystemMetricsInfo
+// type in the model package.
+func (sm SystemMetricsArtifactInfo) Export() model.SystemMetricsArtifactOptions {
+	return model.SystemMetricsArtifactOptions{
+		Format:      sm.Format.Export(),
```

Comment:
Same question about format.

Reply:
This should still be removed since we're no longer using format.

See [this](https://github.com/evergreen-ci/timber/pull/30/files#diff-5921bc410dd2932ed31eef8df216a63eL27)

Reply:
done